cmd/eval.go

```diff
@@ -15,14 +15,18 @@ import (
 var (
 	replacements map[string]string
 	evalCmd      = &cobra.Command{
-		Use:   "eval",
+		Use:   "eval [file]",
```

Comment:
Open to suggestions on how to best expose what argument we are expecting. Looking at the current version of this sub-command on the main branch locally:

```bash
$ ./combo eval --help
...
Usage:
  combo eval [flags]
```

It's not entirely clear that we expect/require an argument to be passed. Adding the "eval [file]" results in the new executable help format:

```bash
$ ./combo eval --help
...
Usage:
  combo eval [file] [flags]
```

Which at least makes an attempt that an argument is required, on top of the example that's provided in the long description of the sub-command. Any thoughts?

Reply:
I like adding the `[file]` here personally. It makes it pretty clear what needs to be passed when combined with the long description like you said.